lib/routes/account.js

```diff
@@ -659,6 +659,14 @@ module.exports = function (
               log.event('verified', { email: account.email, uid: account.uid, locale: account.locale })
               log.increment('account.verified')
               return db.verifyEmail(account)
+                .then(mailer.sendPostVerifyEmail.bind(
+                    mailer,
+                    account.email,
+                    {
+                      acceptLanguage: request.app.acceptLanguage
+                    }
+                  )
+                )
```

Comment:
> I think its a good opportunity to promote Sync either way

Fair point.  Sync is still our big user-value proposition so it makes sense to promot it.

>  their "Firefox Account" whatever that means

Also their "device", *whatever* ***that*** *means*.  I expect we'll do another round of improvements to these once we start getting more and better connected-device information.